src/main/java/net/masterthought/cucumber/json/Element.java

```diff
@@ -8,20 +9,27 @@
 import net.masterthought.cucumber.json.support.StatusCounter;
 import net.masterthought.cucumber.util.Util;
 
+import java.time.LocalDateTime;
+
 public class Element implements Durationable {
 
     // Start: attributes from JSON file report
+    private final String id = null;
     private final String name = null;
     private final String type = null;
     private final String description = null;
     private final String keyword = null;
+    private final Integer line = null;
+    @JsonProperty("start_timestamp")
```

Comment:
Add comment that this field is provided by new cucumber version

Reply:
done